src/index.scss

```diff
@@ -1,29 +0,0 @@
-$fa-font-path: "~font-awesome/fonts";
-@import "~font-awesome/scss/font-awesome";
-
-@import "~@edx/brand/paragon/fonts";
-@import "~@edx/brand/paragon/variables";
-@import "~@edx/paragon/scss/core/core";
-@import "~@edx/brand/paragon/overrides";
-
-@import "~@edx/frontend-component-header/dist/index";
-@import "~@edx/frontend-component-footer/dist/footer";
```

Comment:
Please don't remove these lines. We don't want to remove dependence on bootstrap and paragon, just dependance on bootstrap and paragon variables and functions.